### ISSUE TITLE
Error 'userInfo' trying to play a video of MyTF1

### ISSUE THREAD

**Issue by Simon-G** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1242]
### Describe the bug
Since few day, I have an error trying to start a video from myTF1

### To Reproduce
Steps to reproduce the behaviour:
just start a video from myTF1. For example, problem is reproduced with the replay of "Journal de 13h"

### Your environment
 - [ ] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [x] Raspberry-Pi
 - [ ] Windows
 - [ ] Other

 - Operating system version/name: libreELEC 11.0.3
 - Kodi version: 20.2
 - Catch-up TV & More version: 0.2.39~beta39

### Additional context
Video from france TV for example are working

### Debug log
https://paste.kodi.tv/yahusujine.kodi

**Comment by joaopa00** [comment]
Ton log est trop incomplet pour être utile. Tu dois activer le mode déboggage les chaines de déboggaes (libcurl ffmpeg,  audio et video). Attach le log (et pas copier) ici et non dans un site extérieur si tu veux de l'aide.

**Comment by Simon-G** [comment]
Salut!
Désolé pour ma réponse tardive, j'attendais de voir si d'autres personnes reproduisaient mon problème, apparemment non, et dans ce cas, le problème doit venir de mon environnement... Je voulais poursuivre les tests en réinstallant l'extension, mais je n'ai pas eu le temps de m'en occupé.

Toute aide est bien sur plus que bienvenue! Voici les logs demandés:
[kodi.log](https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/files/13443751/kodi.log)

Merci pour ton retour ;)

**Comment by joaopa00** [comment]
> Repository add-on catchuptvandmore.kodi.krypton.release uses old schema definition for the repository extension point! This is no longer supported, please update your addon to use <dir> definitions.

Je ne sais pas comment tu t'y es pris, mais tu as installé la version Krypton de catchuptvandmore.
CUTVM est cassé chez toi. Je te conseille de desinstaller Kodi ainsi que tous tes réglages et de recommencer une installation  propre de Kodi et de CUTVM.

**Comment by Simon-G** [comment]
Effectivement, maintenant que tu le dis le problème est apparu après que j'ai mis a jour mon libreelec.
Je suis sur LibreElec 11 donc Kodi nexus, j'ai désinstallé CUTVM et l'ai réinstallé en suivant la [doc d'install](https://catch-up-tv-and-more.github.io/fr/installation/), mais le problème reste le même...
Sur la page d'accueil du Github, il est mentionné que 
> Catch-Up TV & More est compatible avec les versions Jarvis (16), Krypton (17), Leia (18) et Matrix (19) de Kodi
Mais aucune mention de Nexus (20)... Est un oubli? CUTVM est il bien compatible avec Nexus?

**Comment by joaopa00** [comment]
Oui CUTVM est compatible avec Nexus. Il doit se passer chez toi qu'il y a um mélange pour Kodi entre Nexus et Krypton. Ce genre de panne si on essaie de comprendre peut prendre très longtemps à résoudre. Le plus simple est de repartir d'un système propre. Il faut desinstaller Kodi ainsi que tous ses fichiers de ocnfiguration.

**Comment by Simon-G** [comment]
Pas de soucis, je comprends! ça faisait un moment que j’envisageai de m'isntaller sur un raspeberry 5, ça sera l'occase. En tous cas j'ai bien comrpsiq ue c'était aps un problème de CUTVM, je clos donc ce ticket.
